phpBB/styles/prosilver/theme/forms.css

```diff
@@ -248,6 +248,10 @@ fieldset.submit-buttons input {
 	margin: 15px 0 2px 0;
 }
 
+.format-buttons .button {
```

Comment:
this belongs in buttons.css also consider doing the same to `.form-buttons select` in this file...

Reply:
also, this needs to be rebase against 3.3.